frontend_tests/node_tests/compose.js

```diff
@@ -1613,3 +1613,20 @@ run_test('create_message_object', () => {
     assert.equal(message.content, 'burrito');
 
 });
+
+run_test('nonexistent_stream_reply_error', () => {
+    set_global('$', global.make_zjquery());
+
+    var shown;
+    var hidden;
+    $("#nonexistent_stream_reply_error").show = () => {
+        shown = _.uniqueId();
+    };
+    $("#nonexistent_stream_reply_error").hide = () => {
+        hidden = _.uniqueId();
```

Comment:
Are we sure uniqueId() always increases?

Reply:
That seems to be the case: https://github.com/jashkenas/underscore/blob/ae037f7c41323807ae6f1533c45512e6d31a1574/underscore.js#L1518